pkg/collector/python/datadog_agent.go

```diff
@@ -295,6 +295,9 @@ var defaultSQLPlanNormalizeSettings = traceconfig.JSONObfuscationConfig{
 		"Relation Name",
```

Comment:
I see some fields that should not be fully excluded, but should pass through an obfuscation layer (i.e. `ObfuscateSQLValues`):

* `Filter`
* `Index Cond`

Reply:
New fields added: d041839969802f77ddeaae96e51bbbd4fe06acb5